cpp/include/legate_dataframe/core/column.hpp

```diff
@@ -417,6 +417,15 @@ class LogicalColumn {
    */
   std::string repr(size_t max_num_items = 30) const;
 
+  /**
+   * @brief Slice the column
+   *
+   * @param slice The Legate slice to the column. Note that legate slices are
+   * inclusive and do not support negative values!  TODO: Check!
```

Comment:
TODO here should be done.

Reply:
Ooops!, to my surprise, it seems to behave like Python slicing.  (I think that may be a doc issue in legate, since I don't think it's obvious in legate land.)